test/fixtures/xml/zeebe-taskListeners.bpmn

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<bpmn:definitions xmlns:bpmn="http://www.omg.org/spec/BPMN/20100524/MODEL" xmlns:bpmndi="http://www.omg.org/spec/BPMN/20100524/DI" xmlns:dc="http://www.omg.org/spec/DD/20100524/DC" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:zeebe="http://camunda.org/schema/zeebe/1.0" xmlns:modeler="http://camunda.org/schema/modeler/1.0" id="Definitions_0qh9wc7" targetNamespace="http://bpmn.io/schema/bpmn" exporter="Camunda Modeler" exporterVersion="5.23.0" modeler:userTaskPlatform="Camunda Cloud" modeler:userTaskPlatformVersion="8.0.0">
+  <bpmn:process id="ZeebePropertiesTest" name="Zeebe Properties Test" isExecutable="true">
+    <bpmn:extensionElements>
+      <zeebe:taskListeners>
+        <zeebe:taskListener eventType="complete" retries="1" type="complete_listener" />
+      </zeebe:taskListeners>
+    </bpmn:extensionElements>
```

Comment:
Let's make sure the fixture matches what we want to implement, i.e. only the User Task should contain the task listeners. Otherwise it's confusing as the properties on the process won't be editable.

```suggestion
```